### PR TITLE
Enable secure protocol

### PR DESCRIPTION
**Pull request by suriyaa**
ping @kallewoof, @luke-jr for ACK

### CODE REVIEW

File: bip-0016.mediawiki [hunk 1]
```diff
@@ -40,7 +40,7 @@ The rules for validating these outpoints when relaying transactions or consideri
...
+These new rules should only be applied when validating transactions in blocks with timestamps >= 1333238400 (Apr 1 2012) <ref>[https://github.com/bitcoin/bitcoin/commit/8f188ece3c82c4cf5d52a3363e7643c23169c0ff Remove -bip16 and -paytoscripthashtime command-line arguments]</ref>. There are transactions earlier than 1333238400 in the block chain that fail these new validation rules. <ref>[https://blockexplorer.com/tx/6a26d2ecb67f27d1fa5524763b49029d7106e91e3cc05743073461a719776192 Transaction 6a26d2ecb67f27d1fa5524763b49029d7106e91e3cc05743073461a719776192]</ref>. Older transactions must be validated under the old rules. (see the Backwards Compatibility section for details).
```
Comment (katesalazar) [inline bip-0016.mediawiki:43]: This is a broken link.

**Review by katesalazar — CHANGES_REQUESTED** [review]
Broken links are replaced with broken links,
causing history noise for no benefit.

File: bip-0016.mediawiki (hunk 1 above)
Comment (suriyaa) [reply bip-0016.mediawiki:43]: @katesalazar Should I remove them all?

**Comment by suriyaa** [comment]
See also related pull request #1250.

**Comment by luke-jr** [comment]
I don't see the need to replace with https everywhere. It's not like the links are privacy or security sensitive...

Also prefer not redirecting blockexplorer.com links to blockchain.com.

But ultimately the authors of each BIP will have to make the call...

**Comment by kallewoof** [comment]
I see this as bloat, personally. I also think we need to keep PRs to per-BIP unless there's a group of them (e.g. the taproot BIPs).

**Comment by katesalazar** [comment]
In 4b795029bf4e I think there are no broken links left to be _fixed_.

**Comment by murchandamus** [comment]
I’m closing this PR as it has multiple merge conflicts and has been inactive for almost three years, the replacement of `http` with `https` seems unnecessary, and the broken links appear to have been fixed via archive links or seem to be working again.

I would suggest that any outstanding issues are addressed with new PRs that focus only on a single document each.

**Pull request closed without merging**